src/components/displaySvg/finalSvgDisplay.scss

```diff
@@ -0,0 +1,67 @@
+.display-page-div{
+    padding: 171px;
+    width: 373px; 
+}
+
+ellipse:hover,rect:hover,path:hover,text:hover,linearGradient:hover,polygon:hover,circle:hover {
+   background-color: yellow;
+   fill: yellow;
+}
+
+ .inline-display-position{
+   padding-top: 71%;
```

Comment:
this is alot of padding, try px

Reply:
its better if you suggest where you can so that we do one thing once, just to type a pixel, how about that ? I have changed it to 40px now.

Reply:
ok cool, if you ngrok we can go through the styling together and I can show you what would be better, we can maybe do that tomorrow

Reply:
or I'll just pull your branch and start it up

Reply:
Okay, I will let you know once I ngrok it !